src/configure.ac

```diff
@@ -1653,8 +1664,16 @@ fi
 AC_MSG_RESULT([$PYTHON_TK_VERSION])
 
 AC_MSG_CHECKING(for site-package location)
-SITEPY=`$PYTHON -c 'import distutils.sysconfig; print(distutils.sysconfig.get_python_lib())'`
+# The sed replacing python3.7 and similiar with python3 is to work
+# with buster, which return version specific path.
+# The sed replacing site-package to work with bullseye and older,
+# which return ../site-packages, not .../dist-packages.
+SITEPY="$($PYTHON -c 'import sysconfig; import re; p1=sysconfig.get_path("platlib"); print(p1); p2=p1.replace("/site-packages", "/dist-packages"); print(p2); print(re.sub("/python3[^/]+/","/python3/", p2))')"
```

Comment:
Again, not knowing anything about linuxcnc, this seems to be completely unaware of PREFIX.

Reply:
Indeed.  I changed the approach to avoid these string replacements.